docs/UserGuide.md

```diff
@@ -179,11 +182,23 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in AA.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Deleting filtered persons : `deleteshown`
+
+Deletes the current filtered list of persons. Requires a `find` command to be run first.
+
+Format: `deleteshown`
+>Note: The application ignores any extraneous parameters as we assume they are typos.
+
+* Deletes all persons in the current filtered list of persons.
+* The list of persons is filtered using the most recent `find` command.
+* The remaining list of persons is shown after the `find` command is executed.
+
```

Comment:
Perhaps here we can include that the user is not intended to use `deleteshown` to delete all the persons but should use the `clear` command instead?